[Bug]: Crash when selecting fields

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### App Version

1.2.1

### Electron Version

15.1.2

### Operating System

Windows

### What happened?

App window turns white and displays nothing after a street name field or zone name field is selected

### Steps To Reproduce

[add_tester.csv](https://github.com/agrc/api-client/files/7462472/add_tester.csv)
Add table
Select a field name

### Relevant log output

react-dom.production.min.js:5193 Error: Action type: undefined is an unknown reducer type. {"action":"UPDATE_FIELDS","payload":"ROADNAME","meta":"street"}
    at Ie (GeocodeContext.js:34)
    at immer.esm.js:949
    at e.produce (immer.esm.js:962)
    at immer.esm.js:947
    at Object.sa [as useReducer] (react-dom.production.min.js:3859)
    at n.useReducer (react.production.min.js:389)
    at use-immer.module.js:18
    at Ge (GeocodeContext.js:39)
    at ia (react-dom.production.min.js:3762)
    at Ga (react-dom.production.min.js:4406)

--- a/src/components/FieldLinker.jsx
+++ b/src/components/FieldLinker.jsx
@@ -48,7 +48,7 @@ const FieldLinker = () => {
         <select
           name="street"
           onChange={(event) =>
-            geocodeDispatch({ action: 'UPDATE_FIELDS', payload: event.target.value, meta: 'street' })
+            geocodeDispatch({ type: 'UPDATE_FIELDS', payload: event.target.value, meta: 'street' })
           }
           value={geocodeContext.data.street}
         >
@@ -60,7 +60,7 @@ const FieldLinker = () => {
         <label htmlFor="zone">Zone Field Name</label>
         <select
           name="zone"
-          onChange={(event) => geocodeDispatch({ action: 'UPDATE_FIELDS', payload: event.target.value, meta: 'zone' })}
+          onChange={(event) => geocodeDispatch({ type: 'UPDATE_FIELDS', payload: event.target.value, meta: 'zone' })}
           value={geocodeContext.data.zone}
         >
           <option>please select a field</option>

